osu.Game/Online/RealtimeMultiplayer/MultiplayerRoomUser.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Game.Users;
+
+namespace osu.Game.Online.RealtimeMultiplayer
+{
+    [Serializable]
+    public class MultiplayerRoomUser : IEquatable<MultiplayerRoomUser>
+    {
+        public readonly long UserID;
+
+        public MultiplayerUserState State { get; set; } = MultiplayerUserState.Idle;
+
+        public User? User { get; set; }
+
+        public MultiplayerRoomUser(in int userId)
```

Comment:
Incoming param is `int` but local field is `long` - not sure if intended.

Reply:
`int` is intended - i don't think we're going to hit `long` user IDs for quite a while longer.